roles/contiv_network/tasks/ovs_cleanup.yml

```diff
@@ -25,9 +25,16 @@
 
 - debug: var=ports
 
+- name: check selinux status
```

Comment:
Reason is to undo whatever we do in ovs.yml file here. How come removing this task, will execute "deny openvswitch_t type in selinux" for redhat when Selinix us enabled ?